src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -195,7 +172,13 @@ export default class HaAutomationActionRow extends LitElement {
                 ></ha-icon-button>
               `
             : ""}
-          <ha-button-menu corner="BOTTOM_START" @action=${this._handleAction}>
+          <ha-button-menu
+            slot="header"
```

Comment:
Same goes here, but IMHO this should be inside the collapsed panel.  You don't need it unless you're working on it.

Reply:
Excellent point. Maybe we can show it only if expanded or move it into the menu itself? But let's explore that in a future PR.